deeptime/decomposition/_score.py

```diff
@@ -3,28 +3,68 @@
 
 import numpy as np
 
-from deeptime.numeric import schatten_norm as _schatten_norm, is_sorted, spd_inv_sqrt
-from ..covariance import CovarianceModel
+from ..numeric import is_sorted, spd_inv_sqrt, schatten_norm
 
 
-class KoopmanModel:
+def vamp_score(koopman_model, r: Union[float, str],
+               covariances_test=None, dim: Optional[int] = None, epsilon: float = 1e-6):
+    """Compute the VAMP score between a covariance-based Koopman model and potentially a
+    test model for cross-validation.
 
-    def __init__(self, koopman_matrix):
-        self.koopman_matrix = koopman_matrix
+    Parameters
+    ----------
+    koopman_model : deeptime.decomposition.CovarianceKoopmanModel
+        The model to score.
+    r : float or str
+        The type of score to evaluate. Can by an floating point value greater or equal to 1 or 'E', yielding the
+        VAMP-r score or the VAMP-E score, respectively. :cite:`vampscore-wu2020variational`
+        Typical choices are:
 
+        *  'VAMP1'  Sum of singular values of the half-weighted Koopman matrix.
```

Comment:
This could be a bit misleading I think. If I understand correctly 'r' can be either a float>1 or str='E', but afterwards at the typical choices it sounds like it could be also 'VAMP1', VAMP2 or VAMPE. I would put at the typical choices the choice of r as well.

Reply:
done!